src/manifold/components/feature-attribution-control-container.js

```diff
@@ -6,10 +6,14 @@ import {
   StyledSlider,
   StyledSelect,
   SelectArrow,
-} from './styled-components';
+} from 'packages/mlvis-common/ui/styled-components';
+import {
+  SegmentFilterPanel,
+  SegmentGroupPanel,
+} from 'packages/mlvis-common/ui/segment-panels';
```

Comment:
Is this `segment-panels` only used for segment filters?
If it is generalizable to be used in other apps, shall we rename it accordingly?